index.js

```diff
@@ -1,143 +1,14 @@
-            genlog.log(bot, c.message.author, {
-              message: `Ran the command \`${cmd}\``,
-              result: res.result,
-              affected: res.affected,
-              awardPoints: true
-            })
-          })
-        } catch (e) {
-          logger.log(bot, {
-            cause: cmd,
-            message: e.message
-          })
-          woofmeow.woofmeow().then((c) => {
-            msg.reply(`an error occurred while processing this command, the admins have been alerted, please try again later.\nHere's your consolation animal image: ${c}`)
-          })
-        }
-      })
-    }
-  }
-})
-
-bot.Dispatcher.on(Events.MESSAGE_REACTION_ADD, (m) => {
-  if (m.user.id !== bot.User.id) {
-    if (m.message === null) {
-      bot.Channels.get(m.data.channel_id).fetchMessages().then(() => {
-        m.message = bot.Messages.get(m.data.message_id)
-      })
-    }
-    Commands['registerVote'].fn(m.message, m.emoji, bot, uvClient, m.user)
-  }
-})
-
-bot.Dispatcher.on(Events.GUILD_MEMBER_UPDATE, (c) => {
-  for (let role of c.rolesAdded) {
-    if (role.id === '268815388882632704') {
-      bot.Channels.get('284796966641205249').sendMessage(`Welcome ${c.member.mention} to the custodians!`)
-    }
-  }
-})
-
-bot.Dispatcher.on(Events.GATEWAY_READY, () => {
-  setInterval(() => {
-    bot.Users.fetchMembers() // Hacky way to cache offline users, #blamelazyloading
-  }, 600000)
-
-  // admin-queue amount playing status
-  r.db('DFB').table('queue').filter(doc => doc('type').match("^(adminMergeRequest|adminReviewDelete)$")).count().run().then(c => bot.User.setGame(c.toString() + " reports in queue"))
-  setInterval(() => {
-    r.db('DFB').table('queue').filter(doc => doc('type').match("^(adminMergeRequest|adminReviewDelete)$")).count().run().then(c => bot.User.setGame(c.toString() + " reports in queue"))
-  }, 60000)
-
-  console.log('Feedback bot is ready!')
-
-  // Autorole!
-  Analytics.roleUsers(bot.Guilds.get(Config.discord.guild), bot)
-  setInterval(() => {
-    Analytics.roleUsers(bot.Guilds.get(Config.discord.guild), bot)
-  }, 3600000) // once an hour
-
-  Commands['initializeTop'].fn(bot, uvClient)
-})
-
-process.on('uncaughtException', (err) => {
-  logger.raven(err.stack || err)
-})
-
-process.on('unhandledRejection', (reason, p) => {
-  if (p !== null && reason !== null) {
-    if (reason instanceof Error) logger.raven(reason)
-    else logger.raven(new Error(`Unhandled promise: ${require('util').inspect(p, {depth: 3})}: ${reason}`))
-  }
-})
-
-bot.Dispatcher.on(Events.DISCONNECTED, (e) => {
-  console.error('Connection to Discord has been lost... Delay till reconnect:', e.delay)
-})
-
-bot.Dispatcher.on(Events.GATEWAY_RESUMED, () => {
-  console.log('Reconnected.')
+Client.Dispatcher.onAny((type, ctx) => {
+  if (Events[type]) Events[type](ctx)
 })
 
-bot.connect({
-  token: Config.discord.token
+Client.connect({
+  token: Config.Discord.token
 })
```

Comment:
These changes + the deletion of the ``queue.js`` file are because of the readme branch being based off a develop branch that was discontinued. Hence all the extra stuff going on in this pr, squashing this should result in the extra weirdness to null itself out.